glean-core/src/lib.rs

```diff
@@ -81,15 +81,31 @@ static GLEAN: OnceCell<Mutex<Glean>> = OnceCell::new();
 /// Get a reference to the global Glean object.
 ///
 /// Panics if no global Glean object was set.
```

Comment:
Is this comment still true after the protections you added?

Reply:
Of course not, missed that. thanks!